sjb/config/common/test_cases/origin_release_install_azure.yml

```diff
@@ -0,0 +1,190 @@
+      parameters:
+        - OS_PUSH_TAG
+        - OS_PUSH_BASE_REGISTRY
+    - type: "script"
+      title: "push the release"
+      repository: "origin"
+      script: |-
+        sudo chmod a+rw /tmp/.docker/config.json
+        DOCKER_CONFIG=/tmp/.docker OS_PUSH_LOCAL=1 OS_PUSH_ALWAYS=1 hack/push-release.sh
+    - type: "script"
+      title: "push the image registry release"
+      repository: "image-registry"
+      script: |-
+        DOCKER_CONFIG=/tmp/.docker OS_PUSH_LOCAL=1 OS_PUSH_ALWAYS=1 hack/push-release.sh
+    - type: "script"
+      title: "push the kubernetes metrics server release"
+      repository: "kubernetes-metrics-server"
+      script: |-
+        DOCKER_CONFIG=/tmp/.docker OS_PUSH_LOCAL=1 OS_PUSH_ALWAYS=1 hack/push-release.sh
+    - type: "script"
+      title: "push the web console release"
+      repository: "origin-web-console-server"
+      script: |-
+        DOCKER_CONFIG=/tmp/.docker OS_PUSH_LOCAL=1 OS_PUSH_ALWAYS=1 hack/push-release.sh
+        sudo rm -rf /tmp/.docker
+
+    - type: "host_script"
+      title: "move secrets to remote host"
+      script: |-
+        # FIXME: Forward Azure credentials from Jenkins host to remote host
+        # rsync --copy-links --omit-dir-times --archive --rsh "ssh -F ./.config/origin-ci-tool/inventory/.ssh_config" /var/lib/jenkins/.gce/* openshiftdevel:/data/src/github.com/openshift/release/cluster/test-deploy/${CLUSTER_PROFILE}/
+        ssh -F ./.config/origin-ci-tool/inventory/.ssh_config openshiftdevel 'sudo chown -R origin:origin-git /data/src/github.com/openshift/release/cluster/test-deploy/${CLUSTER_PROFILE}/'
+        ssh -F ./.config/origin-ci-tool/inventory/.ssh_config openshiftdevel 'sudo chmod -R ug+rwX /data/src/github.com/openshift/release/cluster/test-deploy/${CLUSTER_PROFILE}/'
+
+    - type: "script"
+      title: "provision test cluster"
+      repository: "release"
+      timeout: 2800
+      script: |-
+        # FIXME: Fetch openshift/release branch/fork that we can test now
+        # FIXME: Build openshift-ansible branch/fork we want to use
+        sudo systemctl restart docker
+
+        suffix='origin-${component}'
+        suffix="${suffix}:$( cat /data/src/github.com/openshift/aos-cd-jobs/ORIGIN_TAG )"
+        cd cluster/test-deploy/${CLUSTER_PROFILE}/
+        ../../bin/ansible.sh ansible-playbook \
+          -e "openshift_test_repo=${location_url}" \
+          -e debug_level=5 \
+          -e "oreg_url=${OS_PUSH_BASE_REGISTRY}${OS_PUSH_BASE_REPO}${suffix}" \
+          playbooks/gcp/openshift-cluster/launch.yml \
+          "${evars:-}"
+        cp admin.kubeconfig /tmp/cluster-admin.kubeconfig
+  post_actions:
+    - type: "script"
+      title: "deprovision test cluster"
+      repository: "release"
+      timeout: 900
+      script: |-
+        trap 'exit 0' EXIT
+        cd cluster/test-deploy/${CLUSTER_PROFILE}
+        ../../bin/ansible.sh ansible-playbook playbooks/gcp/openshift-cluster/deprovision.yml
```

Comment:
We do not have a deprovision yet.  This should probably be:
```
az group delete --resource-group <resource group name>
```

Reply:
we do have `deprovision.yml` which deletes resource group, so I think this is fine, but it should be `make down`